locations/tasks.py

```diff
@@ -0,0 +1,47 @@
+from future.standard_library import install_aliases
+install_aliases()  # noqa
+from urllib.parse import urlparse
+
+from celery.task import Task
+from django.conf import settings
+from seed_services_client import IdentityStoreApiClient
+
+from .models import Parish
+
+
+class SyncLocations(Task):
+    """
+    Has a look at all the identity store identities, and ensures that all of
+    the locations assigned to identities appear in the list of locations.
+    """
+    def get_identities(self, client):
+        """
+        Returns an iterator over all the identities in the identity store
+        specified by 'client'.
+        """
+        identities = client.get_identities()
+        while True:
+            for identity in identities.get('results', []):
+                yield identity
+            if identities.get('next') is not None:
+                qs = urlparse(identities['next']).query
+                identities = client.get_identities(params=qs)
```

Comment:
Yes, it pulls the querystring from the `next` url, and gives it as a param to the identity store client to fetch the next page (because the client doesn't support pagination). I'll add a comment for this.